.github/release-please/release-please-config.json

```diff
@@ -76,7 +76,7 @@
     "platform": {
       "component": "platform",
       "release-type": "node",
-      "prerelease": false,
+      "prerelease": true,
```

Comment:
doing this so that creation of platform GitHub Releases don't mark any platform related releases as "latest" (this affects the desktop app download/asset-finding flow with https://archestra.ai)